app/controllers/blogs_controller.rb

```diff
@@ -1,40 +1,61 @@
 class BlogsController < ApplicationController
 
-  before_action :logged_in_user, only: [:new, :edit, :index, :show, :create,]
-
+  before_action :authenticate_user!, except: [:show]
 
   def index
-
+    @blog = Blog.where(user_id: current_user.id)
```

Comment:
Add pagination for this blog. Naming convention should be multiple since it is fetching multiple values.